src/pfe/file-watcher/server/test/functional-test/configs/project.config.ts

```diff
@@ -11,64 +11,99 @@
 /**
  * This file for different project specific (i.e liberty, spring, docker) capabaility configurations.
  */
+
```

Comment:
Can you add a comment here saying why we need a manual reset because if others are looking at the tests, they may not necessarily recognize about the bash scripts messing up build queues at first